pkg/operator/controller/ingress/deployment.go

```diff
@@ -53,6 +53,8 @@ const (
 	RouterSyslogFacilityEnvName  = "ROUTER_LOG_FACILITY"
 	RouterSyslogMaxLengthEnvName = "ROUTER_LOG_MAX_LENGTH"
 
+	RouterContainerMaxLengthEnvName = "ROUTER_LOG_MAX_LENGTH"
```

Comment:
You cannot use the same user-visible value for two different environment variables.